android/build.gradle

```diff
@@ -49,14 +49,15 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.31'
+    implementation "com.github.doublesymmetry:kotlin-audio:0.1.35"
+//    implementation "com.github.DoubleSymmetry:KotlinAudio:v0.1.33"
```

Comment:
We can delete this right?

Reply:
I think this needs to be reverted as this is pointing the RNTP dependency at the locally built `KotlinAudio` project.